src/index.js

```diff
@@ -97,7 +103,10 @@ export type {MenuListPropsType} from './components/list/MenuList';
 export {default as Logo} from './components/logo/Logo';
 export type {LogoPropsType} from './components/logo/Logo';
```

Comment:
@clxandstuff I believe we talked that changing type in logo component would be good

Reply:
Sorry, I misunderstood that it just 'would be good' but we will cover it later😄

Reply:
Logo also exported now :)

Reply:
No worries, I wasn't sure either :)